functions/api/modules/sendgrid.js

```diff
@@ -0,0 +1,23 @@
+const { MailService } = require('@sendgrid/mail');
+
+let sendgrid;
+
+function getSendGrid() {
+  if (sendgrid) return sendgrid;
+  sendgrid = new MailService();
+  sendgrid.setApiKey(process.env.SENDGRID_API_KEY);
```

Comment:
Should make no practical difference, but I think its a good practice to only set the cached`sendgrid` after `setApiKey` is called